src/app/Processor/ProcessorEdit/FiltersEdit/FiltersEdit.tsx

```diff
@@ -52,27 +58,62 @@ const FiltersEdit = (props: FiltersEditProps): JSX.Element => {
       isPlaceholder: true,
     },
     {
-      value: "StringEquals",
-      label: t("processor.stringEquals"),
+      value: FilterType.STRING_EQUALS,
+      label: t("processor.StringEquals"),
+      isPlaceholder: false,
+    },
+    {
+      value: FilterType.STRING_CONTAINS,
+      label: t("processor.StringContains"),
       isPlaceholder: false,
     },
     {
-      value: "StringContains",
-      label: t("processor.stringContains"),
+      value: FilterType.STRING_BEGINS,
+      label: t("processor.StringBeginsWith"),
       isPlaceholder: false,
     },
     {
-      value: "StringBeginsWith",
-      label: t("processor.stringBeginsWith"),
+      value: FilterType.STRING_IN,
+      label: t("processor.StringIn"),
       isPlaceholder: false,
     },
     {
-      value: "ValuesIn",
-      label: t("processor.valuesIn"),
+      value: FilterType.NUMBER_IN,
+      label: t("processor.NumberIn"),
       isPlaceholder: false,
     },
   ];
 
+  const getOptionalTooltipIcon = (
+    filter: EventFilter
+  ): JSX.Element | undefined =>
+    isCommaSeparatedFilterType(filter) ? (
+      <Popover
+        headerContent={t("processor.multipleValues")}
+        bodyContent={t("processor.commaSeparatedValuesTooltip")}
+      >
+        <button
+          type="button"
+          aria-label={t("processor.moreInfoForFilterValues")}
+          onClick={(e): void => e.preventDefault()}
+          aria-describedby="form-group-label-info"
+          className="pf-c-form__group-label-help"
+        >
+          <HelpIcon noVerticalAlign={true} />
+        </button>
+      </Popover>
+    ) : undefined;
+
+  const getOptionalPlaceholder = (filter: EventFilter): string | undefined => {
+    if (filter.type === FilterType.STRING_IN) {
+      return "hello,world,how,are,you";
```

Comment:
how is user intended to put values like:
- "Hello World"
- "Jozef's query"

I mean should we demonstrate non single word strings in the help? Are such supported by backend?

Reply:
Input string is comma separated, so whatever you put between will be considered a filter item.
So, for your example: "Hello World, Jozef's query".
For the backend I think spaces are perfectly fine. However you can try it out with the OpenApi web interface